seed/views/portfoliomanager.py

```diff
@@ -258,15 +288,20 @@ def generate_and_download_template_report(self, matched_template):
 
         # Finally we can download the generated report
         template_report_name = urllib.quote(matched_template["name"]) + ".xml"
-        download_url = "https://portfoliomanager.energystar.gov/pm/reports/template/download/%s/XML/false/%s" % (
-            str(template_report_id), template_report_name
+        sanitized_template_report_name = template_report_name.replace('/', '_')
```

Comment:
This is the fourth and final important addition here: The report template name had forward slashes in it (a date to be precise), and that wasn't being sanitized.  So I added that replacement here.  @nllong do we have a single location in SEED for sanitizing data for urls?  Should I use like urllib escape or whatever it is?